DTDL/dataflow_request.json

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:io:tributech:dataflow:request;1",
+  "@type": "Interface",
+  "extends": ["dtmi:io:tributech:dataflow:base;1"],
+  "displayName": "Request",
+  "description": "Allows to request a dataflow form an external node to the current node.",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "Comment",
+      "schema": "string",
+      "writable": true,
+      "displayName": "Comment",
+      "description": "The comment for the Request. (optional)"
+    },
+    {
+      "@type": "Relationship",
+      "name": "RequestedStreams",
```

Comment:
Streams could be moved to base for Request/Subscription since there is no difference at the moment.